src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1547,6 +1547,13 @@ private static bool NormalizeTaskTypesInPointer(CSharpCompilation compilation, r
                 }
             }
 
+            if (type.ContainsNullableReferenceTypes())
+            {
+                // PROTOTYPE(NullableReferenceTypes): Create attribute
+                // with PEModuleBuilder.SynthesizeNullableAttribute().
+                // See AttributeTests_Nullable.EmitAttribute_Interface.
```

Comment:
Could you add a note to test a mix of tuple names and nullability? The code skeleton looks like it will not handle that.

Reply:
Thanks. Updated extension method and unit test.